### ISSUE TITLE
Waiting on a new release

### ISSUE THREAD

**Issue by sfgetzmd02** [go-git/go-git#179]
There have been many PRs merged in the last 4 months or so and there has not been a release to include them. I've noted the information on the main README stating there were some legal issues and that they have now been resolved. I'm wondering when to expect the next release?

**Comment by nanmu42** [comment]
For souls of curiosity like me, this link may help: https://github.com/src-d/go-git/issues/1295#issuecomment-592965250

> The problem was that in November source{d} went bankrupt, and this repository felt in a legal void. Actually, none of us has merge rights anymore. But don't worry we are figuring out how to tackle this problem, as a hard deadline this will be cleared in mid-March.
> 
> I co-founded a new company, and this company will be the official maintainer of go-git. But we are open to any other IC or company willing to maintain the project with us.
> 
> We will move the repository a new organization, but we need to resolve the legal issues about the repository itself and see if we can move the repository there, or we need to do a hard fork losing all the issues and PRs, but we want to avoid this as much as possible.
> 
> This will be solved in mid-March, since the new company is initiating the activity in the next weeks and our project relies heavily on go-git.

**Comment by pjbgf** [comment]
Closing this in favour of a more recent discussion https://github.com/go-git/go-git/discussions/687.
